content/docs/studio/get-started.md

```diff
@@ -1,31 +1,34 @@
 # Get Started with DVC Studio
 
 Here, we will walk you through a tutorial to use DVC Studio for collaboration on
-your ML projects. You will need access to a Github, Gitlab or Bitbucket account
+your ML projects. You will need access to a GitHub, GitLab or Bitbucket account
 which has access to the Git repositories you want to connect. DVC Studio creates
 views from repositories when you connect to them.
 
 ## What is a view?
 
 A _view_ is an interactive representation of the experiments run in your ML
 project. DVC Studio automatically identifies datasets, metrics and
-hyperparameters in your ML experiments. You can also add custom files with the
-required data (more on this later). Using this data, DVC Studio creates a
-summary view. This includes tables of all the metrics across all your
-experiments. You can also generate plots and compare experiments here.
+hyperparameters in your ML experiments. The data can either be in DVC
+repositories or you can add custom files with the required data. Refer to
+[View settings](/doc/studio/view-settings) to understand the different ways in
+which you can prepare your Git repository for use with DVC Studio. Using this
+data, DVC Studio creates a summary view. This includes tables of all the metrics
```

Comment:
not sure about capitalisation consistency (summary view/summary View/Summary View)

Reply:
Since this is not a keyword, I guess we can leave it at 'summary view'? What do you think?

Reply:
idk, I just noticed you'd been using capital "View" earlier so I assume it should be the same here.

Reply:
` This includes tables` - why tables (plural?). To my mind view == table (with some filters applied).